src/components/Demo/DemoCreate/DemoCreateReview/DemoPreview.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useCallback, useRef, FunctionComponent } from "react";
+import {
+  FrameConnector,
+  HostActions,
+  renderDocument,
+  FrameActions,
+} from "@govtechsg/decentralized-renderer-react-components";
+import { OpenAttestationDocument, utils } from "@govtechsg/open-attestation";
+
+type Dispatch = (action: HostActions) => void;
```

Comment:
move to types? was used in decentralise renderer as well

Reply:
👌